Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RevitRootObjectBuilder.cs

```diff
@@ -53,15 +53,27 @@ CancellationToken cancellationToken
     foreach (var id in objects)
     {
       var el = converterSettings.Current.Document.GetElement(id);
-      if (el != null)
+      if (el == null)
       {
-        revitElements.Add(el);
+        continue;
       }
+
+      if (el.Category == null)
+      {
+        continue;
+      }
+
+      if (!SupportedCategoriesUtils.IsSupportedCategory(el.Category))
+      {
+        continue;
```

Comment:
Would be nice but not sure how much reporting i am talking about
```c#
if (!SupportedCategoriesUtils.IsSupportedCategory(el.Category))
{
  results.Add(new(Status.WARNING, el.UniqueId, el.GetType().Name, null, "Element category is not supported."));
  continue;
}
```

Reply:
legit, ideally we'd report this once per category - i'll see if i can manage that easily